blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -103,7 +103,6 @@ export function PresentationalNav(props) {
 					}`}
 					direction="vertical"
 					justification="start"
-					onClick={closeDrawer}
```

Comment:
With this removed, the variable is now created and passed in the params, but never used so they'll need to be removed if this is no longer needed along with removing it from the calling method.

Reply:
removed!